src/Console/LintCommand.php

```diff
@@ -124,4 +141,16 @@ private function getSeverityLimit(InputInterface $input): int
                 throw new \InvalidArgumentException('Invalid severity limit provided.');
         }
     }
+
+    /**
+     * Check if empty convert option passed
+     *
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @return bool
+     */
+    private function hasEmptyConvertOption(InputInterface $input)
```

Comment:
```suggestion
    private function hasEmptyConvertOption(InputInterface $input): bool
```

Reply:
This function can be deleted. This is not used